internal/slicer/slicer.go

```diff
@@ -113,6 +105,10 @@ func Run(options *RunOptions) (*Report, error) {
 				if sourcePath == copyrightPath && targetPath == copyrightPath {
 					hasCopyright = true
 				}
+			} else if pathInfo.Kind == setup.GeneratePath {
+				// "generate" implies generating new things in a path. Thus, we
+				// do not want anything from the package for GeneratePath.
```

Comment:
Can you reword this a little, I dont understand the `GeneratePath` bit. I think what you are trying to say is that we are generating new files so we are not extracting anything from the package, is that correct?

Reply:
Yup, you are right. But I will reword it.

Reply:
Love it